### DISCUSSION TITLE
Select the inner-content (without \begin and \end)

### DISCUSSION THREAD

**Discussion by Symbol1** [James-Yu/LaTeX-Workshop#3744] [Q&A] [answered]
It appears that `latex-workshop.select-envcontent`
selects the current environment plus the `\begin{...}` and `\end{...}` part.
I wonder if there is a way to select only the inner-content of an environment
(which is anything strictly between `\begin{...}` and `\end{..}`).

More context:
Currently I have the following in my `settings.json`.

```json
    "vim.visualModeKeyBindings":[
        {
            "before": ["a", "e"],
            "commands": [
                "latex-workshop.select-envcontent"
            ]
        },
    ],
```

So when I press `vae`, the vim plugin will call the latex plugin
and the latter will select the environment for me.
I am looking for a way to make `vie` work,
where `i` stands for the inner content.

**Comment by jlelong** [comment, marked as answer]
This is now the default behaviour of the function _LaTeX Workshop: Select the current environment content_. If you want to select the opening and closing statements, use _LaTeX Workshop: Select the current environment_ .

See https://github.com/James-Yu/LaTeX-Workshop/wiki/Environments#Navigating-and-selecting

**Reply by Symbol1** [reply]
Thank you for changing the behavior (non-backward-compatibly) for me 🎉

**Reply by Symbol1** [reply]
![Screenshot 2023-03-26 at 12 10 00](https://user-images.githubusercontent.com/22782566/227798640-3bfea05b-c936-4b26-a1b9-a59226721002.png)

Maybe the first "including" is "excluding"

**Reply by jlelong** [reply]
Thanks for catching the typo
